### DISCUSSION TITLE
[overseer] disabled. run failed: listen tcp 0.0.0.0:9000: bind: address already in use

### DISCUSSION THREAD

**Discussion by FranchukAndrei** [0xJacky/nginx-ui#497] [Q&A]
Installing Nginx-ui on Ubuntu 24.04 LTS (GNU/Linux 6.8.0-39-generic x86_64) using a script `bash <(curl -L -s https://raw.githubusercontent.com/0xJacky/nginx-ui/master/install.sh) install`
After i input in terminal: `nginx-ui -config app.ini`
i get massage:
```
[overseer] disabled. run failed: listen tcp 0.0.0.0:9000: bind: address already in use
INFO    nginx-ui/main.go:20     Nginx configuration directory: /etc/nginx
INFO    nginx-ui/main.go:31     Server exited
```
and if i visit http:\\IP:Port\install
there is only a massage:
```
{
    "message": "not found"
}
```

What am i doing wrong?

**Comment by 0xJacky** [comment]
Once you installed Nginx UI, you don't need to start it by using `nginx-ui -config app.ini` anymore.

And you should visit `http://ip:port` in your brower, it will automatically redirect you to the install page.
